programs/ssl/ssl_client2.c

```diff
@@ -4075,6 +4080,11 @@ int main( int argc, char *argv[] )
 
         mbedtls_printf( " ok\n" );
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && \
+    defined(MBEDTLS_ZERO_RTT) && defined(MBEDTLS_SSL_CLI_C)
+    mbedtls_printf( "reconnect early data status = %d\n", mbedtls_ssl_get_early_data_status( &ssl ) );
```

Comment:
This is in conflict with tests grepping for `early data status = XXX` because that's a suffix of `reconnect early data status = XXX`. How about `early data status, reconnect = XXX`?